resources/provider/migrations/timescale/24_v0.10.16.up.sql

```diff
@@ -4,3 +4,37 @@
 -- Resource: sns.subscriptions
 ALTER TABLE IF EXISTS aws_sns_subscriptions DROP CONSTRAINT aws_sns_subscriptions_pk;
 ALTER TABLE IF EXISTS aws_sns_subscriptions ADD CONSTRAINT aws_sns_subscriptions_pk PRIMARY KEY (cq_fetch_date,endpoint,owner,protocol,arn,topic_arn);
+
+-- Resource: mq.brokers
+ALTER TABLE IF EXISTS aws_mq_brokers
+    DROP CONSTRAINT aws_mq_brokers_pk;
+ALTER TABLE IF EXISTS "aws_mq_brokers"
+    RENAME COLUMN broker_id TO id;
+ALTER TABLE IF EXISTS aws_mq_brokers
+    ADD CONSTRAINT aws_mq_brokers_pk PRIMARY KEY (cq_fetch_date, account_id, id);
+ALTER TABLE IF EXISTS aws_mq_broker_configurations
+    DROP CONSTRAINT aws_mq_broker_configurations_pk;
+ALTER TABLE IF EXISTS aws_mq_broker_configurations
+    ADD CONSTRAINT aws_mq_broker_configurations_pk PRIMARY KEY (cq_fetch_date, cq_id);
```

Comment:
I am not sure if dropping a PK on a hypertable and recreating will work out of the box. I would double check that with a TSDB flavor db.